oci/spec_opts.go

```diff
@@ -33,3 +34,51 @@ func WithHostname(name string) SpecOpts {
 		return nil
 	}
 }
+
+// WithEnvironmentVariables appends environment variables
+func WithEnvironmentVariables(environmnetVariables []string) SpecOpts {
```

Comment:
`WithEnv` is fine. No need to be overly verbose.

Reply:
Fixed.